engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/AncientDataMotionStrategy.java

```diff
@@ -466,13 +466,17 @@ protected Answer migrateVolumeToPool(DataObject srcData, DataObject destData) {
             s_logger.error(errMsg);
             answer = new Answer(command, false, errMsg);
         } else {
+            s_logger.info("Sending MIGRATE_COPY request to node " + ep);
```

Comment:
Do we need all logs here with INFO level, they seem like they should be TRACE or DEBUG

Reply:
addressing in upcoming commit.